src/components/Garden/ModalFlows/CreateProposalScreens/AddProposal.js

```diff
@@ -61,11 +61,16 @@ const AddProposalPanel = React.memo(({ setProposalData }) => {
 
   const fundingMode = formData.proposalType === FUNDING_PROPOSAL
 
-  const forumRegex = new RegExp(connectedGarden.forumURL)
+  const forumRegex = new RegExp(
+    String.raw`(${connectedGarden.forumURL.replace(
+      /[-/\\^$*+?.()|[\]{}]/g,
+      '\\$&'
+    )}|https:\/\/discord\.com\/channels\/)`
+  )
```

Comment:
Escaping the forumURL to avoid getting regex from Onboarding.

Should I put it all in consts btw?
Or its possible to understand?